jsonnet/windows_shortcuts.jsonnet

```diff
@@ -214,5 +214,42 @@ local k = import 'lib/karabiner.libsonnet';
            k.input('f4', ['control']),
            k.outputKey('w', ['command']),
            k.condition('if', ['^com\\.google\\.Chrome$'])),
+    // App Shortcuts
+    k.rule('Open Finder with Cmd+1',
+           k.input('1', ['command']),
+           k.outputShell('open -b com.apple.finder'),
+           k.condition('unless', bundle.standard)),
+    k.rule('Open first pinned app with Cmd+2',
+           k.input('2', ['command']),
+           k.runDockedApp('0'),
+           k.condition('unless', bundle.standard)),
+    k.rule('Open second pinned app with Cmd+3',
+           k.input('3', ['command']),
+           k.runDockedApp('1'),
+           k.condition('unless', bundle.standard)),
+    k.rule('Open third pinned app with Cmd+4',
+           k.input('4', ['command']),
+           k.runDockedApp('2'),
+           k.condition('unless', bundle.standard)),
+    k.rule('Open fourth pinned app with Cmd+5',
+           k.input('5', ['command']),
+           k.runDockedApp('3'),
+           k.condition('unless', bundle.standard)),
+    k.rule('Open fifth pinned app with Cmd+6',
+           k.input('6', ['command']),
+           k.runDockedApp('4'),
+           k.condition('unless', bundle.standard)),
+    k.rule('Open sixth pinned app with Cmd+7',
+           k.input('7', ['command']),
+           k.runDockedApp('5'),
+           k.condition('unless', bundle.standard)),
+    k.rule('Open seventh pinned app with Cmd+8',
+           k.input('8', ['command']),
+           k.runDockedApp('6'),
+           k.condition('unless', bundle.standard)),
+    k.rule('Open eighth pinned app with Cmd+9',
+           k.input('9', ['command']),
+           k.runDockedApp('7'),
+           k.condition('unless', bundle.standard)),
```

Comment:
Let's move this block up to line 191 to keep the alphanumerics together.

Reply:
Makes sense!